static/css/_button.scss

```diff
@@ -0,0 +1,13 @@
+button{
```

Comment:
unless we are implementing new components we shouldn't have these scss files, it's a bit confusing, move the styles to base or elsewhere